demo/src/bench.ts

```diff
@@ -13,36 +15,47 @@ export const sleep = (ms: number): Promise<void> => {
 }
 
 async function main() {
-  await Cord.init({ address: 'ws://127.0.0.1:9944' })
-  const wsProvider = new WsProvider('ws://127.0.0.1:9944')
-  const api = await ApiPromise.create({ provider: wsProvider })
-
+  const networkAddress = 'wss://sparknet.cord.network'
+  Cord.ConfigService.set({ submitTxResolveOn: Cord.Chain.IS_IN_BLOCK })
+  await Cord.connect(networkAddress)
+  const api = Cord.ConfigService.get('api')
   // Step 1: Setup Identities
-  const Alice = Cord.Identity.buildFromURI('//Alice', {
-    signingKeyPairType: 'sr25519',
-  })
-  const Bob = Cord.Identity.buildFromURI('//Bob', {
-    signingKeyPairType: 'sr25519',
-  })
+  const Alice = Crypto.makeKeypairFromUri(
+    '//Sparknet//1//Demo',
+    'sr25519'
+  )
+  console.log(
+    `👩🏻  Alice (${Alice.type}): ${Alice.address}`
+  )
+  const Bob = Crypto.makeKeypairFromUri(
+    '//Sparknet//1//Demo',
```

Comment:
```suggestion
    '//Sparknet//2//Demo',
```

Bob and Alice are same entity here, and hence the issue IMO. Check out after this change.